tests/test_report.py

```diff
@@ -418,6 +423,21 @@ def test_generate_html_report_metadata_json(self):
             self.expected_report_metadata_json
         )
 
+    def test_generate_html_report_metadata_json_metadata_report_not_run_yet(self):
+        validation_result = {}
+        validation_result.update({'vcf_check': {'pass': PROCESS_NOT_RUN_YET}})
+        validation_result.update({'evidence_type_check': {'pass': PROCESS_NOT_RUN_YET}})
+        validation_result.update({'assembly_check': {'pass': PROCESS_NOT_RUN_YET}})
+        validation_result.update({'fasta_check': {'pass': PROCESS_NOT_RUN_YET}})
+        validation_result.update({'metadata_check': {'pass': PROCESS_NOT_RUN_YET}})
+        validation_result.update({'sample_check': {'pass': PROCESS_NOT_RUN_YET}})
```

Comment:
You can just set all of these in the variable declaration, i.e.
```
validation_result = {
    'vcf_check': {'pass': PROCESS_NOT_RUN_YET},
    'evidence_type_check': ...
}
```

Reply:
updated